docs/user/hosting.rst

```diff
@@ -1,28 +1,71 @@
-We support a number of important features that you would expect for a documentation host.
+Hosting features provided by Read the Docs
+==========================================
 
-.. contents:: Contents
-   :local:
-   :depth: 1
+This article provides a quick overview and reference to how documentation is *hosted* on Read the Docs.
+Hosting is optimized with documentation in mind and a number of special features are provided,
+some of which may be customized.
+
+.. seealso::
+
+   :doc:`/reference/features`
+     For a full list of Read the Docs features, see the features reference page.
+
+Overview of hosting features
+----------------------------
 
 Subdomain support
------------------
+  Every project has a subdomain that is available to serve its documentation based on it's :term:`slug`.
+  If you go to ``<slug>.readthedocs.io``, it should show you the latest version of your documentation,
+  for example https://docs.readthedocs.io.
+  For :doc:`/commercial/index` the subdomain looks like ``<slug>.readthedocs-hosted.com``.
 
-Every project has a subdomain that is available to serve its documentation based on it's :term:`slug`.
-If you go to ``<slug>.readthedocs.io``, it should show you the latest version of your documentation,
-for example https://docs.readthedocs.io.
-For :doc:`/commercial/index` the subdomain looks like ``<slug>.readthedocs-hosted.com``.
+Custom domain support
+  Documentation projects may also define their own domains.
+  A project may define multiple domains,
+  as well as define which is the *canonical domain*.
 
-.. seealso::
+  ⏩️ :doc:`Read more </custom-domains>`
+
+Multiple documentation versions
+  The built-in URL structure of Read the Docs is pre-defined to allow for multiple versions and translations to be hosted.
+  You can also switch off this mechanism with :doc:`Single-Version projects </single_version>`.
 
-   :doc:`/custom-domains`
-      More information on using a custom domain.
+  ⏩️ :doc:`Read more </automatic-redirects>`
+
+Redirect support
+  Projects may define their own custom URL redirects rules that trigger in the HTTP layer.
+
+  ⏩️ :doc:`Read more </user-defined-redirects>`
 
 Content Delivery Network (CDN)
-------------------------------
+  Documentation projects are by nature static HTML pages and assets.
+  Hence contents are delivered through a speedy distributed cloud proxy.
+
+  ⬇️ `Read more <Content Delivery Network (CDN) and caching>`__
+
+Sitemaps
+  Sitemaps are generated and hosted automatically,
+  improving search engine crawling.
+
+  ⬇️ `Read more <Sitemaps>`__
+
+Custom 404s
+  A 404 page is provided by default. It may be replaced with a projects own 404 page.
+
+  ⬇️ `Read more <Custom Not Found (404) pages>`__
+
+Automatic and custom robots.txt
+  `robots.txt`_ files allow you to customize how your documentation is indexed in search engines.
+  We automatically generate one for you, but you can also customize it.
+
+  ⬇️ `Read more <Custom robots.txt>`__
```

Comment:
Does this work?

Reply:
Nope, using `:ref:<hosting:...>` instead :+1: